DogeNet.DAL/stylecop.json

```diff
@@ -0,0 +1,15 @@
+﻿{
+  // ACTION REQUIRED: This file was automatically added to your project, but it
```

Comment:
Why does this comment appear here?

Reply:
It was created automatically to specify how to mount the settings file. Since it is no longer needed, I will delete the comment